pkg/audit/filter.go

```diff
@@ -69,3 +69,11 @@ func (aw *wrapWriter) Hijack() (net.Conn, *bufio.ReadWriter, error) {
 	}
 	return nil, nil, errors.New("the ResponseWriter doesn't support the Hijacker interface")
 }
+
+func (aw *wrapWriter) CloseNotify() <-chan bool {
+	return aw.ResponseWriter.(http.CloseNotifier).CloseNotify()
```

Comment:
Please do type assertion for `http.CloseNotifier`, `http.Flusher` and `http.Hijacker`

Reply:
updated